src/httpserver.cpp

```diff
@@ -222,7 +222,7 @@ static bool InitHTTPAllowList()
     std::string strAllowed;
     for (const CSubNet& subnet : rpc_allow_subnets)
         strAllowed += subnet.ToString() + " ";
-    LogPrint("http", "Allowing HTTP connections from: %s\n", strAllowed);
+    LogPrint("Allowing HTTP connections from: %s\n", strAllowed);
```

Comment:
I think this and the other should be `LogPrintf` as the first argument is a `sprintf`-style string.